src/structure_new/src/implicit/4C_structure_new_impl_generic.cpp

```diff
@@ -223,7 +223,7 @@ void NOX::Nln::PrePostOp::IMPLICIT::Generic::runPostIterate(const ::NOX::Solver:
 {
   double step = 0.0;
   const bool isdefaultstep = get_step(step, solver);
-  const int num_corrs = get_number_of_modified_newton_corrections(solver);
+  const int num_corrs = 0;
```

Comment:
If `num_corrs` is always zero now, can it maybe removed entirely (also from `run_post_iterate()`)?

Reply:
I didn't had a detailed look into all of these functions, I just replicated the behavior of the status quo without having to access the deleted parameter list.

Reply:
`num_corrs` was always zero even before my changes.

Reply:
It only has 5 occurrences throughout the code base. I'd love to see it removed, since we now know for sure, that it does not have any effect.

Reply:
Done.